python/lsst/analysis/tools/actions/plot/elements/plotInfoElement.py

```diff
@@ -0,0 +1,146 @@
+
+        bandText = ""
+        for band in plotInfo["bands"]:
+            bandText += band + ", "
+        bandsText = f", Bands: {bandText[:-2]}"
+        infoText = f"\n{run}{datasetsUsed}{tableType}{dataIdText}{bandsText}"
+
+        # Find S/N and mag keys, if present.
+        snKeys = []
+        magKeys = []
+        selectionKeys = []
+        selectionPrefix = "Selection: "
+        for key, value in plotInfo.items():
+            if "SN" in key or "S/N" in key:
+                snKeys.append(key)
+            elif "Mag" in key:
+                magKeys.append(key)
+            elif key.startswith(selectionPrefix):
+                selectionKeys.append(key)
+        # Add S/N and mag values to label, if present.
+        # TODO: Do something if there are multiple sn/mag keys. Log? Warn?
+        newline = "\n"
+        if snKeys:
+            infoText = f"{infoText}{newline if magKeys else ', '}{snKeys[0]}{plotInfo.get(snKeys[0])}"
+        if magKeys:
+            infoText = f"{infoText}, {magKeys[0]}{plotInfo.get(magKeys[0])}"
+        if selectionKeys:
+            nPrefix = len(selectionPrefix)
+            selections = ", ".join(f"{key[nPrefix:]}: {plotInfo[key]}" for key in selectionKeys)
+            infoText = f"{infoText}, Selections: {selections}"
+
+        return infoText
+
+    def __call__(self, ax: Axes, figureInfo: Mapping[str, Any], **kwargs) -> Axes:
+        return self.makeElement(ax, figureInfo, **kwargs)
+
+    def makeElement(self, ax: Axes, figureInfo: Mapping[str, Any], **kwargs) -> Axes:
+        """Draw a text box that displays useful information about the plot.
+
+        Parameters
+        ----------
+        ax : `~matplotlib.axes.Axes`
+            Axes to plot on.
+        figInfo: `dict`
+            A dictionary containing information about the figure on which the
+            element is being placed.  It should have the following keys:
+
+            ``"figsize"``
+                The width and height of the figure in inches
+                (`tuple` of two `float`s)
+            ``"dpi"``
+                The dpi of the figure (`float`)
+
+        Returns
+        -------
+        ax : `~matplotlib.axes.Axes`
+            Updated axes.
+        """
+        plotInfo = kwargs["plotInfo"]
+        plotInfoText = self._buildPlotInfoText(plotInfo)
+
+        dpi_scale_trans = transforms.Affine2D().scale(figureInfo["dpi"])
+        trans = ax.transAxes + transforms.ScaledTranslation(0.0, -0.28, dpi_scale_trans)
+
+        ax.set_axis_off()
+        ax.text(0, 1.0, plotInfo["plotName"], fontsize=7, ha="left", va="bottom")
+        ax.text(0, 1.0, plotInfoText, fontsize=6, alpha=0.6, ha="left", transform=trans)
+
+        # QUERY: Other plotElements return "data", but "data"
+        #        isn't passed to this plotElement "ax" here is a
+        #        placeholder, since it needs to return something.
+        #        Should we use "None"?
+        return ax
```

Comment:
We don't necessarily need to pass data/anything out, do we? If we do need to pass something out, perhaps the parsed plot info would be a good data point to have?

Reply:
But if nothing, this could just be a method which edits in-place.